### ISSUE TITLE
Discussion: Recruiter Credits

### ISSUE THREAD

**Issue by JOHNNYGAZER** [OpenThrone/OpenThrone#276]
One thing that was great back in DT Beta was the ability to "over-click" and build up a certain amount of recruiter credits per day. The max number of recruits per day was something like 500, but you could do 550 clicks per day to build up your credits in case you are not able to access the game for a day or more. This was great since it built up so slowly that only really dedicated players could take a "vacation" from recruiting and regular players could miss a day here and there without losing out on population.

What does everyone think about a system like this in OT?

**Comment by uaktags** [comment]
(using same limits for illustration)
So 500 clicks today would give you 500 citz, but then 501-550th click today would just increment a carry over allotment that we reward at the start/end of a day?
Then tomorrow, you are rewarded those 50 carry overs, would that mean that you can get 500 additional citizens on top of it + 50 carried over again?

**Comment by JOHNNYGAZER** [comment]
The 50 carry-over would accumulate and automatically "cash in" at the end of the day if you didn't do your clicks for the day.
Ex:
Day one = 550 clicks completed, 500 citizens gained w/ 50 in carry-over bank
Day two = 550 clicks completed, 500 citizens gained w/ 50 additional in carry-over bank to total 100
Day three = 550 clicks completed, 500 citizens gained w/ 50 additional in carry-over bank to equal 150
Day four = forget/too busy/unable to do the days clicks. When DT time hits midnight between day four and five, the 150 carry-over clicks are "cashed in" from the bank granting 150 citizens. Carry-over bank reset to zero.

Benefits:
- (for the game) If the intent is to put ads on the clicker, this would serve up to 50 additional ads per day to the user. 
- (for the user) This makes it possible to lessen the blow of missing a day of clicks or miss a day without any impact to growth.

I could see scaling this with housing upgrades or something as well. At the start you can have a max carry-over bank of 500, clicks and each upgrade expands this cap by one full day worth of clicks to 1000, 1500, etc.

**Comment by uaktags** [comment]
Interesting, so an evergrowing bank of citizens (up to some sort of limit)
So on day 4, say you did 400 clicks, would 100 out of the 150 carry overs apply (bringing you to your 500 daily) and leaving you with 50 carry overs? Then day 5, if you do nothing else, it'll cash the remaining carry overs in

**Comment by JOHNNYGAZER** [comment]
Correct, even if you did some, but not all clicks, it would pull from the bank up to the daily limit. This is also nice for the days where I remember to recruit at 23:40 and can't get them all in on time or if I accidentally close the page with clicks still remaining.

**Comment by uaktags** [comment]
Yea im not against it. I'd like to look at how to get the limit to be tied to something though, either Fortifications or House level.

We'd need to instill a daily citizens limit (like the 500), tweak the per-User limit (currently set to 5), and then implement the bank. 
Atleast that's what i'm thinking from a highlevel

**Comment by JOHNNYGAZER** [comment]
I will say, I do think that the way the recruiting limit works now where it scales up with the number of users may be a good thing to keep (at least in my opinion) since it gives newer players a minor catchup mechanic while not giving a disadvantage to the bigger players. That probably warrants a discussion in and of itself.

**Comment by uaktags** [comment]
Definitely further discussion but this still feels like the right place for how intertwined they are, right now I have it in the updated patch to remove IDLE users that have not logged in for some months now, which will reduce the player count and available clicks pretty drastically. With a reduced player count, the auto-recruiter and 5x per user really helps ensure that players get their max potential, because otherwise they MAY NOT get either the full daily limit due to lack of players. On the flip side, having more users than the daily limit, it calls for the question of why allow 5x per user in internal clicks.

In my head, if we have a max recruits per day to be tied to the player count with 5x per user, then there's no real reason to have the carryover. The carry over helps for when you don't click daily, but we already have a catch-up tool built in by having such high limits.

However, if there's a max-perDay limit set to an arbitrary number that scales with your Fort/House, then you never have the ability to catch up as new, and you're at a disadvantage if you miss a few days, which is where the credits come in.
